public/docs/ts/latest/guide/user-input.jade

```diff
@@ -1,34 +1,37 @@
 include ../_util-fns
 
 :marked
-  When the user clicks a link, pushes a button, or enters text
-  we want to know about it. These user actions all raise DOM events.
-  In this chapter we learn to bind to those events using the Angular
+  User actions such as clicking a link, pushing a button, and entering
+  text raise DOM events.
+  This chapter explains how to bind to those events using the Angular
   event binding syntax.
 
   Run the <live-example></live-example>.
 
 :marked
   ## Binding to user input events
 
-  We can use [Angular event bindings](./template-syntax.html#event-binding)
+  You can use [Angular event bindings](./template-syntax.html#event-binding)
   to respond to [any DOM event](https://developer.mozilla.org/en-US/docs/Web/Events).
 
-  The syntax is simple. We surround the DOM event name in parentheses and assign a quoted template statement to it.
-  As an example, here's an event binding that implements a click handler:
+  The syntax consists of surrounding the DOM event name in parentheses and assigning a quoted template statement to it.
+  The following example shows an event binding that implements a click handler:
 +makeExample('user-input/ts/app/click-me.component.ts', 'click-me-button')(format=".", language="html")
 
 <a id="click"></a>
 :marked
-  The `(click)` to the left of the equal sign identifies the button's click event as the **target of the binding**.
-  The text within quotes on the right is the **template statement** in which we
-  respond to the click event by calling the component's `onClickMe` method. A [template statement](./template-syntax.html#template-statements) is a subset
-  of JavaScript with restrictions and a few added tricks.
+  The `(click)`, to the left of the equals sign, identifies the button's click event as the **target of the binding**.
+  The text in quotes, to the right of the equals sign,
+  is the **template statement**, which responds
+  respond to the click event by calling the component's `onClickMe` method.
 
-  When writing a binding we must be aware of a template statement's **execution context**.
+  NOTE: A [template statement](./template-syntax.html#template-statements) is a subset
```

Comment:
That's not how we format notes. You should be able to find an example to follow by grepping other .jade files for "Note:".